pkg/plugins/golang/v3/plugin.go

```diff
@@ -26,7 +27,7 @@ import (
 const pluginName = "go" + plugins.DefaultNameQualifier
 
 var (
-	supportedProjectVersions = []config.Version{cfgv3alpha.Version}
+	supportedProjectVersions = []config.Version{cfgv3alpha.Version, cfgv3.Version}
```

Comment:
IMO as described above. The goal of this PR is only to store the data in the PROJECT file. Then, a follow up would be responsible to make the project-version 3 stable by replacing/renaming v3-alpha per v3 only.